Python/Tests/Core.UI/EnvironmentUITests.cs

```diff
@@ -57,9 +57,7 @@ public class EnvironmentUITests {
             return app.SelectDefaultInterpreter(
                 PythonPaths.Python37 ?? PythonPaths.Python37_x64 ??
```

Comment:
in other place, I could see that you start searching 64bit first then 32bit. however here we are using 32bit first. why?

Reply:
It's very likely "because it doesn't matter". In a lot of tests we just wanted to ensure that a version of Python 3 (or 2) is active, but without having to force contributors into a certain configuration.

Reply:
There is no official rule on 32bit vs 64bit, though I prefer 64bit over 32bit.

Reply:
how about having a single approach? it would let us debug possible issues easier since we will use same CPU architecture through all test cases.

Reply:
It would take a decent amount of work to go through all the tests and change it, so I'm not sure if the cost/benefit analysis is worth it at the moment. Moving forward, we can make it such that all tests should default to 64bit and then 32bit (or vice versa). Whenever someone changes an area of code, they can update the tests to 64bit/32bit. How does that sound?

Reply:
By single approach, do you mean always running the tests on either 32bit or 64bit, but not both?

Reply:
I meant "64bit first and 32bit" like your logic. it would let us run the same tests on different configuration based on what is available. also let us test both cases based on machine config.

Reply:
Agreed. Let's follow this guideline form now on.